package/MDAnalysis/coordinates/NAMDBIN.py

```diff
@@ -0,0 +1,131 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+"""NAMDBIN files format --- :mod:`MDAnalysis.coordinates.NAMDBIN`
+================================================================================
+
+Read/Write coordinates in NAMD_ double-precision binary file (suffix "coor" or "namdbin").

```

Comment:
Please add a link to the definition of the file format. 

If there is no public documentation then please add the file format description to the docs here. 

We need to have a record of what the code is designed to parse.

Reply:
When you link to http://www.ks.uiuc.edu/Research/namd/2.12/ug/node11.html#SECTION00061500000000000000 then make the link text ‘ namdbin file format’ and not just NAMD so that it is clear that this is the official specification.

Reply:
Also add a versionadded 1.0.0 please.